e2e/tests/company/invoices/create.spec.ts

```diff
@@ -313,4 +314,32 @@ test.describe("invoice creation", () => {
     await page.reload();
     await expect(page.getByText("This invoice includes rates above your default")).not.toBeVisible();
   });
+
+  test("supports decimal quantities for hours", async ({ page }) => {
```

Comment:
```suggestion
  test("supports decimal quantities", async ({ page }) => {
```
Minor, but would be confusing since we only support it for _non_-hourly tasks. Would be good to adjust the phrasing below too, just to prevent future confusion.